.travis.yml

```diff
@@ -15,11 +15,17 @@ install:
   - pipenv install --dev --deploy
 
 script:
-  - make build_and_test
+  - pwd
+  - ls -la
```

Comment:
again, we don't need this left in do we?

Reply:
At the moment I'm battling Travis trying to investigate why there's issues. Once it's fixed I'll remove all this stuff

Reply:
Removed it now